src/pages/filter/ui/panels/function-panel/components/gene-region/gene-region.tsx

```diff
@@ -65,6 +66,22 @@ export const GeneRegion = observer(() => {
     return () => filterStore.resetStatFuncData()
   }, [])
 
+  const toggleNotMode = () => {
+    geneRegionStore.setCurrentMode(ModeTypes.Not)
+    filterStore.setChanging(true)
+  }
+
+  const onChangeInput = (e: any) => {
+    geneRegionStore.setLocusValue(e.target.value)
+    validateValue(e.target.value)
+    filterStore.setChanging(true)
```

Comment:
you must check if there is any value or the value is not the same as initial. Relates to all alike cases

Reply:
Done